dlrover/python/diagnosis/inferencechain/inferenceoperator/operator.py

```diff
@@ -29,7 +26,7 @@
 
 
 def get_training_failure_operators():
```

Comment:
Is this replaced by the new diagnotician?

Reply:
Yes. It will be removed in the next pr. This pr focuses on the new design.